### ISSUE TITLE
Block Webshell creation for Servers - Incorrect/Missing OS support

### ISSUE THREAD

**Issue by Hekel1989** [MicrosoftDocs/microsoft-365-docs#13137]
Hi,
![image](https://github.com/MicrosoftDocs/microsoft-365-docs/assets/28776601/109a10ec-2409-4e24-b003-aa3d00700c1f)
the new ASR Rule "Block Webshell creation for Servers" seems to list only Windows 11 in the supported OS:

- Is Windows 11 actually a supported OS? The rule seems to indicate it's a Server-focused ASR.
- What Server OS are supported? According to the current list, none.

Many Thanks

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 71a17626-d89b-a06f-9cc6-308b80a84acc
* Version Independent ID: 71a17626-d89b-a06f-9cc6-308b80a84acc
* Content: [Attack surface reduction rules reference](https://learn.microsoft.com/en-us/microsoft-365/security/defender-endpoint/attack-surface-reduction-rules-reference?view=o365-worldwide#asr-rules-supported-configuration-management-systems)
* Content Source: [microsoft-365/security/defender-endpoint/attack-surface-reduction-rules-reference.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/security/defender-endpoint/attack-surface-reduction-rules-reference.md)
* Service: **microsoft-365-security**
* Sub-service: **mde**
* GitHub Login: @Dansimp
* Microsoft Alias: **dansimp**

**Comment by yogkumgit** [comment]
@Dansimp Please help us in resolving this issue. Thanks

**Comment by denisebmsft** [comment]
https://github.com/MicrosoftDocs/microsoft-365-docs-pr/pull/23092

**Comment by niwelton** [comment]
Yes, this should not say win11. I did a series of updates last week which should resolve this?
